sql/expression/function/aggregation/window/last_value.go

```diff
@@ -84,6 +85,11 @@ func (f *LastValue) Type() sql.Type {
 	return f.Child.Type()
 }
 
+// CollationCoercibility implements the interface sql.CollationCoercible.
+func (f *LastValue) CollationCoercibility(ctx *sql.Context) (collation sql.CollationID, coercibility byte) {
+	return sql.GetCoercibility(ctx, f.Child)
```

Comment:
Doesn't this already happen through Expressioner and sql.GetCoercibility?

Reply:
Indeed it does! So for some of these, since they were explicitly defining the collation coercibility in the last iteration of the PR, I did a simpler conversion to the new optional format by just appending the child. Determining which one had multiple children that weren't important, or one child that was important would have ballooned the conversion time astronomically, so I'm sure there are more instances like these. It doesn't hurt though, so I plan to leave these for now.

If, later on, we add another expression in this function for some reason, then the correct expression to check would still be this specific child. So there's potential upside with little downside.